test/test_convergence.jl

```diff
@@ -46,6 +46,19 @@ function generate_profiles(FT; uf_type = UF.BusingerType())
     return profiles_sfc, profiles_int, param_set
 end
 
+
+function compute_dse_diff(thermo_params, ts_int, ts_sfc, p_int, p_sfc)
```

Comment:
`p` stands for `profile`? It might be good to change the name as `p` looks like pressure.

Reply:
Changed now to `prof_sfc` and `prof_int`.